mk/uvmt/dsim.mk

```diff
@@ -18,14 +18,15 @@
 #
 ###############################################################################
 #
-# DSIM-specific Makefile for the CV32E40P "uvmt_cv32" testbench.
```

Comment:
Do not include - outdated

Reply:
Not the case.  DSIM is still very much in use and the DSIM-specific Makefile is still needed.

Reply:
What I meant to say here is that the 40P-comment in this line is not really useful, as it is a general makefile for all the 40*-cores

Reply:
disregard this - I got the add&removed order swapped here